pkg/lib/v0_2_0/shipyard.go

```diff
@@ -0,0 +1,40 @@
+package v0_2_0
+
+///// v0.2.0 Shipyard Spec ///////
+
+// Shipyard describes a shipyard specification according to Keptn spec 0.2.0
+type Shipyard struct {
+	ApiVersion string           `json:"apiVersion" yaml:"apiVersion"`
+	Kind       string           `json:"kind" yaml:"kind"`
+	Metadata   ShipyardMetadata `json:"metadata" yaml:"metadata"`
+	Spec       ShipyardSpec     `json:"spec" yaml:"spec"`
+}
+
+// ShipyardMetadata describes Shipyayrd metadata
+type ShipyardMetadata struct {
+	Name string `json:"name" yaml:"name"`
+}
+
+// ShipyardSpec consists of any number of stages
+type ShipyardSpec struct {
+	Stages []Stage `json:"stages" yaml:"stages"`
+}
+
+// Stage defines a stage by its name and list of task sequences
+type Stage struct {
+	Name      string     `json:"name" yaml:"name"`
+	Sequences []Sequence `json:"sequence" yaml:"sequence"`
```

Comment:
Should be `json:"sequences" yaml:"sequences"`

Reply:
Fixed it.